sections/header.liquid

```diff
@@ -500,7 +500,7 @@
             {%- if section.settings.logo != blank -%}
               {%- assign logo_alt = section.settings.logo.alt | default: shop.name | escape -%}
               {%- assign logo_height = section.settings.logo_width | divided_by: section.settings.logo.aspect_ratio -%}
-              {{ section.settings.logo | image_url: width: section.settings.width | image_tag:
+              {{ section.settings.logo | image_url: width: 500 | image_tag:
```

Comment:
Here I'm changing it as the width is a max of `250` which means we're never giving the logo a chance to be a `500` wide file.